docs/index.md

```diff
@@ -58,6 +58,29 @@ import 'virtual:group-icons.css'
 export default Theme
 ```
 
+## Usage Example
+<pre style="background-color:var(--vp-code-block-bg);padding:24px;border-radius:8px;">
```

Comment:
We can use ````md to wrap the code instead of `pre` tag. Like this:

https://github.com/vuejs/vitepress/blob/main/docs/en/guide/markdown.md?plain=1#L675